lib/std/c/freebsd.zig

```diff
@@ -17,7 +17,7 @@ pub const sf_hdtr = extern struct {
 pub extern "c" fn sendfile(
     out_fd: fd_t,
     in_fd: fd_t,
```

Comment:
Can you fix these parameter names too? Also in c/darwin.zig

Reply:
done